examples/agentInteractionsDemo/app.js

```diff
@@ -175,13 +175,56 @@ new Vue({
         const usersApi = new platformClient.UsersApi();
         const qualityApi = new platformClient.QualityApi();
         const conversationsApi = new platformClient.ConversationsApi();
+        const externalContactsApi = new platformClient.ExternalContactsApi();
         Vue.prototype.$usersApi = usersApi;
         Vue.prototype.$qualityApi = qualityApi;
         Vue.prototype.$conversationsApi = conversationsApi;
+        Vue.prototype.$externalContactsApi = externalContactsApi;
 
         let authenticated = false;
         let agentUserId = "";
 
+        function getCustomerParticipant(conv) {
+            let newConv = conv;
+            newConv.customer = newConv.participants.find((part) => {
+                return part.purpose === "customer" || part.purpose === "external";
+            }) || { name: "Unknown" };
+            return newConv.customer
+        }
+
+        async function getCustomerName(customer){
+            if (customer.externalContactId){
+                return externalContactsApi.getExternalcontactsContact(customer.externalContactId)
+                    .then((externalContact) => {
+                        const name = `${externalContact.firstName} ${externalContact.lastName}`;
+                        return Promise.resolve(name)
+                    })
+            } else{
+                if(!customer.name){
+                    return Promise.resolve("Unknown")
+                } else{
+                    return Promise.resolve(customer.name)
+                }
+            } 
+        }
+
+        async function getEvalConversationsData(evaluations){
+
+            const evalConversations = await Promise.all(
+                evaluations.map(eval => conversationsApi.getConversation(eval.conversation.id).then(
+                    async (conv) => {
+                        conv.customer = getCustomerParticipant(conv);
+                        const name = await getCustomerName(conv.customer);
+                        conv.customer.name = name;
+                        return await Promise.resolve(conv);
+                    }
+                ).catch((err) => {
+                    console.log(`Error: ${err}`);
+                }))
+            );
+            return await Promise.resolve(evalConversations);
```

Comment:
Same here, since you're in an async function, you can just return and that will resolve the promise.